### ISSUE TITLE
subscribing to program changes for system program only is hella boring

### ISSUE THREAD

**Issue by nathanleclaire** [workbenchapp/solana-workbench#146]
we should probably also subscribe to token program at least ... maybe some of the other built in ones ... and it would be hella clutch if we would also allow user to enumerate their own custom programs to subscribe to

**Comment by SvenDowideit** [comment]
so instead of subscribing to one program's changes, enable multi-select? that does make sense :D

**Comment by nathanleclaire** [comment]
Yep exactly actually didn’t @Bluskript implement that already ha ha

**Comment by Bluskript** [comment]
I didn't unfortunately, I built the UX assuming multi-sleect was possible but only realized later that it only allowed one selection.

**Comment by nathanleclaire** [comment]
Ah word well yea we’ll look to guide things in that direction in follow ups

**Comment by SvenDowideit** [comment]
I'd love to have a list (all?) programs that are on the selected validator, with linked info from https://apr.dev

this selection list would be reused by the amman validator rc file generation UX to enable a user to import programs to their local validator
